test/testcafe/spec/EnrollProfileView_spec.js

```diff
@@ -204,23 +194,22 @@ test.requestHooks(requestLogger, EnrollProfileSignUpAllBaseAttributesMock)('All
     division: 'Division',
     department: 'Department',
     managerId: 'Manager ID',
-    manager: 'Manager',
+    //manager: 'Manager', //TODO: Found multiple elements with the text of: /Manager/
   };
 
-  Object.keys(formFieldToLabel).forEach(async (formField) => {
```

Comment:
forEach does not wait for all promises to resolve, so remain test code will be executed after test end.
This leads to false positive result of current test (see "Manager") and false negative (flacky) results of next tests

Reply:
Good catch